src/libraries/System.Console/tests/ManualTests/ManualTests.cs

```diff
@@ -96,6 +90,34 @@ static string RenderKeyChord(ConsoleKeyInfo key)
             }
         }
 
+        public static IEnumerable<object[]> GetKeyChords()
+        {
+            yield return MkConsoleKeyInfo('\x01', ConsoleKey.A, ConsoleModifiers.Control);
+            yield return MkConsoleKeyInfo('\x01', ConsoleKey.A, ConsoleModifiers.Control | ConsoleModifiers.Alt);
+            yield return MkConsoleKeyInfo('\r', ConsoleKey.Enter, (ConsoleModifiers)0);
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // windows will report '\n' as 'Ctrl+Enter', which is typically not picked up by Unix terminals
+                yield return MkConsoleKeyInfo('\n', ConsoleKey.Enter, ConsoleModifiers.Control);
+            }
+            else
+            {
+                yield return MkConsoleKeyInfo('\n', ConsoleKey.J, ConsoleModifiers.Control);
```

Comment:
Should we 'translate' this in `ConsolePal` so it becomes as `ConsoleKey.Enter` instead of `ConsoleKey.J`?

Reply:
I'm intentionally removing that translation since you can't really use the enter key to input `\n`, at least not in the terminal emulators that I tested.